packages/components/src/internal/components/chart/ChartFieldRangeScaleOptions.tsx

```diff
@@ -0,0 +1,157 @@
+            if (selected === 'automatic') {
+                onScaleChange(field.name, 'min', undefined);
+                onScaleChange(field.name, 'max', undefined);
+                scale_ = { ...scale_, min: undefined, max: undefined };
+            }
+            setScale(scale_);
+        },
+        [field.name, onScaleChange, scale, setScale]
+    );
+
+    const onScaleMinChange = useCallback(
+        (event: ChangeEvent<HTMLInputElement>) => {
+            setScale(prev => ({ ...prev, min: event.target.value }));
+        },
+        [setScale]
+    );
+
+    const onScaleMaxChange = useCallback(
+        (event: ChangeEvent<HTMLInputElement>) => {
+            setScale(prev => ({ ...prev, max: event.target.value }));
+        },
+        [setScale]
+    );
+
+    const onScaleRangeBlur = useCallback(() => {
+        if (invalidRange) return;
+        onScaleChange(field.name, 'min', parseFloat(scale.min?.toString()));
+        onScaleChange(field.name, 'max', parseFloat(scale.max?.toString()));
+    }, [field.name, onScaleChange, scale.max, scale.min, invalidRange]);
+
+    return (
+        <div className="field-option-icon">
+            <OverlayTrigger
+                overlay={
+                    <Popover id="chart-field-option-popover" placement="bottom">
+                        {children}
+                        <div className="field-option-radio-group">
+                            <label>Scale</label>
+                            <RadioGroupInput
+                                formsy={false}
+                                name="scaleTrans"
+                                onValueChange={onScaleTransChange}
+                                options={scaleTransOptions}
+                            />
+                        </div>
+                        <div className="field-option-radio-group">
+                            <label>Range</label>
+                            <RadioGroupInput
+                                formsy={false}
+                                name="scaleType"
+                                onValueChange={onScaleTypeChange}
+                                options={scaleTypeOptions}
+                            />
+                        </div>
+                        {scale.type === 'manual' && (
+                            <div className="chart-builder-scale-range-inputs">
+                                <input
+                                    className="chart-builder-field-footer-input"
+                                    name="scaleMin"
+                                    onBlur={onScaleRangeBlur}
+                                    onChange={onScaleMinChange}
+                                    placeholder="Min"
+                                    type="number"
+                                    value={scale.min ?? ''}
+                                />
+                                <span>&nbsp;&nbsp;-&nbsp;</span>
```

Comment:
We could skip the `&nbsp;` and use the `chart-builder-field-footer-input` class to apply spacing.

Reply:
yup, that works. done. FYI, this was unchanged code from my PR (just moved / refactored).